.github/workflows/unit_tests.yml

```diff
@@ -129,6 +137,7 @@ jobs:
         EB_VERBOSE: 1
         EASYBUILD_MODULE_SYNTAX: ${{matrix.module_syntax}}
         TEST_EASYBUILD_MODULE_SYNTAX: ${{matrix.module_syntax}}
+        LC_ALL: ${{matrix.lc_all}}
```

Comment:
Now you're always setting `$LC_ALL`. Setting it to an empty value is *not* the same as not setting it at all.

So please make this conditional below (if `${{matrix.lc_all}}` is not empty, use it to define `$LC_ALL` via `export`

Reply:
I think this is fine. I tested `locale` and `LC_ALL= locale` and got the same output. I haven't found any information telling me otherwise but found https://www.baeldung.com/linux/locale-environment-variables
```
To reset it, we could export an empty value to LC_ALL:

$ export LC_ALL=
```

As this solution is the cleanest, easiest (code wise), I'd suggest to keep it

Reply:
OK, good enough then.